src/mpyl/utilities/jenkins/runner.py

```diff
@@ -19,6 +20,36 @@
 from . import Pipeline
 
 
+def stream_utf_8_logs(self, interval=0):
+    """
+    Return generator which streams parts of text console.
+    Workaround for https://github.com/pycontribs/jenkinsapi/pull/843
```

Comment:
low faith in this getting merged any time soon? 😄

Reply:
Yeah, but in any case I don't want to wait